pkgs/by-name/gs/gscan2pdf/package.nix

```
@@ -74,18 +80,11 @@ perlPackages.buildPerlPackage rec {
     SubOverride
   ]);
 
-  postPatch =
-    let
-      fontSubstitute = "${liberation_ttf}/share/fonts/truetype/LiberationSans-Regular.ttf";
-    in
-    ''
-      # Required for the program to properly load its SVG assets
-      substituteInPlace bin/gscan2pdf \
-        --replace "/usr/share" "$out/share"
-
-      # Substitute the non-free Helvetica font in the tests
-      sed -i 's|-pointsize|-font ${fontSubstitute} -pointsize|g' t/*.t
-    '';
+  postPatch = ''
+    # Required for the program to properly load its SVG assets
+    substituteInPlace bin/gscan2pdf \
+      --replace "/usr/share" "$out/share"
+  '';
```

```suggestion
  # Required for the program to properly load its SVG assets
  postPatch = ''
    substituteInPlace bin/gscan2pdf \
      --replace-fail "/usr/share" "$out/share"
  '';
```

Should be showing a warning about deprecated behavior. https://github.com/NixOS/nixpkgs/issues/356002